File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.js
let res = 1
let list = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");

input.forEach(i => res *= parseInt(i));
res = res.toString();

for(i of res) list[i] += 1

list.forEach(i => console.log(i));

File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.cc
#include <string>
#include <iostream>
using namespace std;

int main() {
    int a,b,c,l[10]={0,0,0,0,0,0,0,0,0, 0};
    string r;
    
    cin >> a;
    cin >> b;
    cin >> c;

    r = to_string(a*b*c);

    for(int i=0; i<r.size(); i++) {
        l[r[i] - 48]++;
    }

    for(int i=0; i<10; i++) {
        cout << l[i] << "\n";
    }

    return 0;
}